Cambios en el capítulo 5, Administración de Dependencias

3rn3st0 opened the pull request:
Revisado, corregido/cambiado el contenido del capítulo 5 del documento principal. Espero por nuevas instrucciones. Asignen el trabajo que consideren necesario hacer sin problemas.

3rn3st0 committed:
Trabajando con UTF-8, traducido para chequeo. #19.

3rn3st0 committed:
Trabajando con UTF-8, correcciones hechas según sugerencias de @oyepez003. #19.

3rn3st0 committed:
Capítulo 5 Administración de Dependencias #18

Se editaron los 3 documentos pertenecientes al capítulo 5 del documento
principal. Se les hicieron pequeños cambios en cuanto a corrección de la
estructura gramatical y algunos pocos errores de ortografía. NO se hizo
traducción de ninguna índole.

* 04-01-01-Dependency-Management.md

* 04-02-01-Composer-and-Packagist.md

* 04-03-01-PEAR.md

---
Issue: resolve  #18

Status: Resuelto

Related: n/r

3rn3st0 committed:
Traducción de la página Functional Programming #40

Se hizo la traducción completa de la página de Functional Programming,
se queda en espera de revisión y observaciones para hacer los cambios
que se requieran para cerrar el issue.

---
Issue: #40

Status: En progreso

Related: n/r

3rn3st0 committed:
Traducción de la página Functional Programming #40

Correcciones gramaticales y de ortografía.

---
Issue: #40

Status: En progreso

Related: n/r

oyepez003 commented on pages/Functional-Programming.md:
```diff
@@ -1,86 +1,73 @@
-Closure is an anonymous function that can access variables imported from the outside scope without using any global
-variables. Theoretically, a closure is a function with some arguments closed (e.g. fixed) by the environment when it is 
-defined. Closures can work around variable scope restrictions in a clean way.
+Un cierre es una función anónima que puede acceder a las variables importadas fuera del ámbito sin usar variables globales. En teoría, un cierre es una función con algunos argumentos cerrados (por ejemplo, fijo) por el entorno cuando este ha sido definido. Los cierres pueden evitar las restricciones de ámbito de variables de una manera limpia.
```
@3rn3st0 Creo que debemos dejar la palabra Closure porque "cierre" no lo veo bien como traducción... Ej.: "Un Closure es una función anónima..." Aplica para todos los demás párrafos donde se refieren a la palabra Closure o Closures.

ernestochapon replied:
Apenas hoy vengo a ver este comentario. Lo corrijo mañana y retomo las traducciones.

3rn3st0 committed:
Cambiado "cierre" por el original: "closure" #40

Por sugerencia de @oyepez, se cambió el término traducido "cierre" por
el original en inglés "closure". De esa manera el texto tiene más
sentido y no pierde el contexto técnico.

3rn3st0 reviewed (commented):
Se hizo el cambio que se sugirió y ahora el texto mantiene un contexto técnico cuyo sentido lógico es mejor que el de la traducción original.

oyepez003 commented on pages/Functional-Programming.md:
```diff
@@ -1,86 +1,73 @@
-Closure is an anonymous function that can access variables imported from the outside scope without using any global
-variables. Theoretically, a closure is a function with some arguments closed (e.g. fixed) by the environment when it is 
-defined. Closures can work around variable scope restrictions in a clean way.
+Un cierre es una función anónima que puede acceder a las variables importadas fuera del ámbito sin usar variables globales. En teoría, un cierre es una función con algunos argumentos cerrados (por ejemplo, fijo) por el entorno cuando este ha sido definido. Los `closures` pueden evitar las restricciones de ámbito de variables de una manera limpia.
```
Sigue estando la palabra "cierre". @3rn3st0

oyepez003 commented on pages/Functional-Programming.md:
```diff
@@ -1,86 +1,73 @@
-Each filter function in the family accepts only elements greater than some minimum value. Single filter returned by 
-`criteria_greater_than` is a closure whith `$min` argument closed by the value in the scope (given as an argument when 
-`criteria_greater_than` is called).
+Cada función de filtro en la familia aceptará sólo los elementos mayores que el valor mínimo. El filtro sencillo retornado por `criterio_mayor_que` es un cierre con un argumento `$min` cerrado por el valor en el ámbito (dado como argumento cuando `criterio_mayor_que` es llamado).
```
Sigue estando la palabra "cierre".

oyepez003 commented on pages/Functional-Programming.md:
```diff
@@ -1,86 +1,73 @@
-Early binding is used by default for importing `$min` variable into the created function. For true closures with late
-binding one should use a reference when importing. Imagine a templating or input validation libraries, where closure is 
-defined to capture variables in scope and access them later when the anonymous function is evaluated.
+El primer enlace es usado por defecto para importar la variable `$min` en la función. Para `closures` verdaderos con uniones tardías debe usarse una referencia cuando se importen. Imagina una plantilla o bibliotecas de validación de entradas, donde se define el cierre para capturar variables en el ámbito y acceder luego, cuando se evalúe la función anónima.
```
Sigue estando la palabra "cierre". @3rn3st0

3rn3st0 commented:
Ya me monto en esas correcciones. Cambie **"cierres"** y obvié **"cierre"**. En un rato subo los cambios.

3rn3st0 committed:
Se sustituyó la palabra "cierre" por "closure"

Se cambiaron todas las referencias a **"cierre"** dentro del texto,
sustituyéndolas por **closure**.

3rn3st0 commented:
¡Listo, correcciones hechas!

3rn3st0 committed:
1er borrador traducción Issue #28

Traducciones preliminares de documentos del Capítulo 14: Virtualización
    * 13-01-01-Virtualization.md y
    * 13-02-01-Vagrant.md
